doc/custom.properties

```diff
@@ -1,2 +1,2 @@
 manual.name=addon-serial_modem-__VERSION__
-booktitle=Serial Modem Add-on __VERSION__
+booktitle=Serial Modem __VERSION__
```

Comment:
@divipillai Is this something that should be changed?
And what about line one?

Reply:
This refers to the Bundle name of the doc-set - https://docs.nordicsemi.com/bundle. Asset tracker template also has add-on in it. I think we can keep it as is.

Reply:
Reverted this change.